UserDetailsClient/UserDetailsClient.Core/App.xaml.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+using Microsoft.Identity.Client;
+using UserDetailsClient.Core.Features.LogOn;
+
+[assembly: XamlCompilation(XamlCompilationOptions.Compile)]
+namespace UserDetailsClient.Core
+{
+	public partial class App : Application
+    {
+
+        public static string ApiEndpoint = "https://fabrikamb2chello.azurewebsites.net/hello";
+
+        public App ()
+		{
+            InitializeComponent();
+
+            DependencyService.Register<B2CAuthenticationService>();
```

Comment:
Fancy :). Maybe add a comment on how to achieve this without dependency injection, as DI is a pretty advanced conceptg.

Reply:
advanced? ^_^

comments added...